lib/cocoapods/installer/analyzer/pod_variant.rb

```diff
@@ -24,16 +28,20 @@ def root_spec
 
         # Initialize a new instance from its attributes.
         #
-        # @param [Array<String>] specs       @see #specs
-        # @param [Platform] platform         @see #platform
-        # @param [Bool] requires_frameworks  @see #requires_frameworks?
+        # @param [Array<Specification>] specs @see #specs
+        # @param [Platform] platform          @see #platform
+        # @param [Bool] requires_frameworks   @see #requires_frameworks?
         #
         def initialize(specs, platform, requires_frameworks = false)
           self.specs = specs
+          self.testspecs = []
```

Comment:
why not add this to the initializer?

Reply:
hm it might actually be OK now that I look at it.